### DISCUSSION TITLE
How to get rid of mixed content warning in Firefox because wrong called logo

### DISCUSSION THREAD

**Discussion by c1972** [YOURLS/YOURLS#3420] [Q&A]
Hello,

I have Yourls running public with the Sleeky Frontend: [https://cc10.de/](https://cc10.de/)
( https://github.com/Flynntes/Sleeky )

When viewing my Yourls in Firefox the browser shows a mixed content warning.

![ff1](https://user-images.githubusercontent.com/112912128/188528413-6a2a0234-bf1d-47b2-814b-3e9de41bf772.JPG)

It seems to be a logo called without SSL or http instead of https.

![ff2](https://user-images.githubusercontent.com/112912128/188528412-20593c11-229a-4ef1-a1b3-bdedf731027f.JPG)

Settings for this logo are in https://cc10.de/frontend/config.php

This line there only accepts a relative path like
define('logo', '/frontend/assets/img/logo-84x84-4.png');

When I try it with an absolute path URL it results in http://https://cc10.de...

What can I do to call this logo with SSL ?

Thank you!

Chris

**Comment by dgw** [comment]
Sleeky is not maintained by us. You should report this problem there and ask its developer(s) to help.

The current [code for this line of HTML](https://github.com/Flynntes/Sleeky/blob/f7978d9d426e4373a02785c5c008920f4bec7a96/sleeky-frontend/index.php#L101) in that repo looks like it should work fine, though. Your linked frontend page exhibits very odd behavior; while both the problem line and [this later one](https://github.com/Flynntes/Sleeky/blob/f7978d9d426e4373a02785c5c008920f4bec7a96/sleeky-frontend/index.php#L122) use `YOURLS_SITE`, the later one outputs `https://` as expected. The same constant can't have two different values in the same execution pass, so I wonder if your code is outdated or somehow modified.

**Reply by c1972** [reply]
Hello dgw,

thanks for your answer!

The file you mention is a different one.

I mean the config.php in /frontend/ and there its line 19:
https://github.com/Flynntes/Sleeky/blob/f7978d9d426e4373a02785c5c008920f4bec7a96/sleeky-frontend/frontend/config.php#L19

I found no other place or discussion to ask my question, sorry. Thank you for helping me!

Chris

**Reply by dgw** [reply]
That is where you _set_ `logo`. The first line I linked is where the value is _used_.

I should point out that the Sleeky repo you initially linked has [a Help section](https://github.com/Flynntes/Sleeky#help) in its readme offering three different ways to contact the author about Sleeky itself. Chances are he does not monitor these discussions. 🙂

**Comment by c1972** [comment]
Thank you dgw!

I will try to reach someone there. This was my first ever post on github. Pls excuse me I am not familiar with this here (and not with php).

If you don't know php like me... it's difficult to fine tune code like yourls. And its probably difficult to find someone willing to help with free software...

Chris
